prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -0,0 +1,33 @@
+groups:
+- name: host.alerts
+  rules:
+  - alert: HostWithRunningVMsNotResponding
+    expr: |
+    vrops_hostsystem_stats{statkey="summary_running_VMs_number"} > 0 and 
+    ON (hostsystem) vrops_hostsystem_properties{propkey="runtime_connectionState: notResponding"} and 
+    ON (hostsystem) vrops_hostsystem_properties{propkey="runtime_maintenanceState: notInMaintenance"} and 
+    ON (hostsystem) vrops_hostsystem_properties{propkey="runtime_powerState: PoweredOn"}
```

Comment:
Please use lowercase `on`.
I would recommend explicit checking of the value, since this is what the value is going to be.
```
on (hostsystem) vrops_hostsystem_properties{propkey="runtime_connectionState: notResponding"} == 0
```

Reply:
Recommendation implemented